stanford_news.module

```diff
@@ -34,6 +40,27 @@ function stanford_news_preprocess_block(&$variables) {
       $variables['#attached']['library'][] = 'stanford_news/newsletter_signup';
     }
   }
+  if (!empty($variables['elements']['content']['#cache']['contexts'])) {
+    if (in_array('route.menu_active_trails:news-topics', $variables['elements']['content']['#cache']['contexts'])) {
+      $variables['#attached']['library'][] = 'stanford_news/news_list';
+    }
+  }
```

Comment:
@boznik attaching a library by a cache context seems irregular. Can you explain this choice?